confbot_msgs/action/MoveCommand.action

```diff
@@ -0,0 +1,10 @@
+# Goal
+int32 duration  # in seconds
+float32 linear_velocity  # in meter/second
+float32 angular_velocity  # in rad/second
+---
+# Result
+float32 distance_traveled  # in meter
+---
+# Feedback
+float32 distance_traveled  # in meter
```

Comment:
could we add the time elapsed in the feedback as well ? that would give a better estimate of the progression of the task